vm/src/stdlib/itertools.rs

```diff
@@ -196,13 +195,19 @@ mod decl {
             vm: &VirtualMachine,
         ) -> PyResult {
             let start = match start.into_option() {
-                Some(int) => int.as_bigint().clone(),
-                None => BigInt::zero(),
+                Some(num) => vm.new_pyobj(num),
+                None => vm.new_pyobj(0),
             };
```

Comment:
```rust
let start: PyObjectRef = start.into_option().unwrap_or_else(|| vm.new_pyobj(0));
```

Reply:
Sorry. I was wrong. I'll fix it. Thanks for the suggestion.